sympy/polys/domains/complexfield.py

```diff
@@ -87,6 +90,18 @@ def from_ZZ_gmpy(self, element, base):
     def from_QQ_gmpy(self, element, base):
         return self.dtype(int(element.numerator)) / int(element.denominator)
 
+    def from_GaussianIntegerRing(self, element, base):
+        return self.dtype(int(element.x), int(element.y))
+
+    def from_GaussianRationalField(self, element, base):
+        x = element.x
+        y = element.y
+        return (self.dtype(int(x.numerator)) / int(x.denominator) +
+                self.dtype(0, int(y.numerator)) / int(y.denominator))
```

Comment:
Why two `self.dtype` calls?

Reply:
We need to divide by two different denominators and I want the division to be computed by CC because it has a configurable precision. If we divide the ints directly and then pass them to dtype the division would be calculated using float (i.e. 53 bit precision).

Reply:
I see. That is a good point.